### PR TITLE
gnome apps: update & fix license [ci skip]

### PR DESCRIPTION
**Pull request by k4leg**
List of updates:
- [x] at-spi2-atk -> 2.34.1
- [x] atk -> 2.35.1
- [x] baobab -> 3.34.0
- [x] cairomm -> 1.15.4
- [x] dconf -> 0.34.0
- [x] dconf-editor -> 3.34.2
- [x] eog-plugins -> 3.26.4
- [x] eolie -> 0.9.63
- [x] epiphany -> 3.34.2
- [x] evince -> 3.34.2
- [x] evolution -> 3.34.2
- [x] evolution-data-server -> 3.34.2
- [x] file-roller -> 3.35.1
- [x] gdl -> 3.34.0
- [x] glib -> 2.63.3
- [x] glib-networking -> 2.63.2
- [x] glibmm -> 2.63.1
- [x] gmime3 -> 3.2.5
- [x] gnome -> fix
- [x] gnome-2048 -> 3.34.4
- [x] gnome-boxes -> 3.34.2
- [x] gnome-calculator -> fix
- [x] gnome-colors-icon-theme -> fix license
- [x] gnome-control-center -> 3.34.2
- [x] gnome-desktop -> 3.34.2
- [x] gnome-disk-utility -> 3.34.0
- [x] gnome-epub-thumbnailer -> 1.6
- [x] gnome-flashback -> 3.34.2
- [x] gnome-games -> 3.34.1
- [x] gnome-getting-started-docs -> 3.34.1
- [x] gnome-keyring -> 3.34.0
- [x] gnome-maps -> 3.34.2
- [x] gnome-mime-data -> fix license
- [x] gnome-music -> 3.34.2
- [x] gnome-passbook -> 0.8
- [x] gnome-photos -> 3.34.0
- [x] gnome-podcasts -> 0.4.7
- [x] gnome-screenshot -> 3.34.0
- [x] gnome-session -> 3.34.2
- [x] gnome-settings-daemon -> 3.34.2
- [x] gnome-shell -> 3.34.2
- [x] gnome-shell-extensions -> 3.34.2
- [x] gnome-sound-recorder -> 3.34.0
- [x] gnome-subtitles -> 1.6
- [x] gnome-themes-standard-metacity -> 3.27.90
- [x] gnome-usage -> 3.33.2
- [x] gnome-user-docs -> 3.34.1
- [x] gnome-video-effects -> 0.5.0
- [x] gnome-weather -> 3.34.0
- [x] gnote -> 3.34.1
- [x] graphene -> 1.10.0
- [x] grilo -> 0.3.10
- [x] gspell -> 1.8.2
- [x] gtk-doc -> 1.32
- [x] gtk-vnc -> 1.0.0
- [x] gtkmm -> 3.24.2
- [x] gtksourceview4 -> 4.4.0
- [x] gvfs -> 1.43.2
- [x] jsonrpc-glib -> 3.34.0
- [x] libchamplain -> 0.12.20
- [x] libgdata -> 0.17.11
- [x] libsecret -> 0.19.1
- [x] libsigc++ -> 3.0.0
- [x] libsoup -> 2.68.3
- [x] mutter -> 3.34.2
- [x] nautilus -> 3.34.2
- [x] network-manager-applet -> 1.8.24
- [x] orca -> 3.34.1
- [x] pangomm -> 2.43.1
- [x] rygel -> 0.38.3
- [x] shotwell -> 0.31.0
- [x] template-glib -> 3.34.0
- [x] tepl -> 4.3.1
- [x] vala -> 0.47.1
- [x] valadoc -> 0.47.1
- [x] vte3 -> 0.59.0

Compiled on:
* **x86_64-musl**: _Performed | 76/76 packages_
* **aarch64-musl**: _Performed | 72/72 packages_
**Non-cross-compiling**: `eog-plugins`, `gnome-flashback`, `gnome-maps` & `shotwell`

### CODE REVIEW

**Comment by Hoshpak** [comment]
Errors out with 

```
meson.build:67:0: ERROR: Dependency "champlain-0.12" not found, tried pkgconfig
```

**Comment by k4leg** [comment]
Thanks! Unfortunately, I will fix this later(1-2 d.), because now I can not do it.

**Comment by k4leg** [comment]
`gthumb` will be placed in another post so as not to delay updating other packages. I can’t verify this package, since it depends on `brasero-devel`, which depends on `cdrtools`, which is not cross-compiled. If you can provide this package (`cdrtools`) for _aarch64_, I will be grateful to you.
The rest of the packages that Travis did not compile (due to time limitations), I will check for _x86_64_ and _aarch64_.

**Comment by k4leg** [comment]
All packages successfully compiled on _x86_64_ and _aarch64_.
`gthumb` moved to another post.
`gnome-authenticator` will be moved to another post.
`gnome-podcasts` and `fractal` are removed because it depends on _openssl_.

**Comment by k4leg** [comment]
Probably a dumb question, but why doesn't it compile? No, I know that because of nautilus, but I normally compile on [x86_64](http://dpaste.com/0ZNX0B5) and [aarch64](http://dpaste.com/0KJE0XV).

**Comment by xtraeme** [comment]
You should test musl as well

**Comment by k4leg** [comment]
@xtraeme Thanks! `nautilus` does not compile on x86_64-musl. Now I will try to fix it.

**Comment by k4leg** [comment]
It seems like everything is fine.

**Comment by Hoshpak** [comment]
@void-linux/pkg-committers 

unfortunately I don't have time to review this right now.

**Comment by Johnnynator** [comment]
203492c9b63176734a0a2548c4f7e397c8768766 53652bbb128306d7b5d17a044d2f5da8a826aac9 Don't remove them, try to patch them. I'm currently compiling everything in this branch to get a look into it myself.

**Comment by Johnnynator** [comment]
I do have updates for both these packages in my branch. Pls test if they run correctly. https://github.com/Johnnynator/void-packages/commits/gnome

File: srcpkgs/evolution/template [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 configure_args="-DENABLE_AUTOAR=OFF -DENABLE_LIBCRYPTUI=OFF -DENABLE_GTKSPELL=OFF
  -DENABLE_TEXT_HIGHLIGHT=OFF -DENABLE_PST_IMPORT=OFF
  -DENABLE_WEATHER=$(vopt_if gir ON OFF)"
 hostmakedepends="gnome-doc-utils gobject-introspection intltool itstool
  pkg-config"
 makedepends="NetworkManager-devel clutter-gtk-devel enchant2-devel
- evolution-data-server-devel gspell-devel gtkhtml-devel libcanberra-devel
- $(vopt_if gir libgweather-devel) libnotify-devel webkit2gtk-devel ytnef-devel
- gnome-desktop-devel"
+ evolution-data-server-devel>=$version gspell-devel gtkhtml-devel
```
Comment (Johnnynator) [inline srcpkgs/evolution/template]: Don't use version constraints in makedepends, the build system of the package you are building should be the one checking it. Also it is breaking quite a few assumptions in `xbps-src` (e.g. breaks `show-build-deps`). They should only be used for runtime dependencies.

**Comment by Johnnynator** [comment]
Gnome Calculator seems to fail cross compiling for me without having `build_helper=gir`
```diff
diff --git a/srcpkgs/gnome-calculator/template b/srcpkgs/gnome-calculator/template
index a46c116339..edea271b2f 100644
--- a/srcpkgs/gnome-calculator/template
+++ b/srcpkgs/gnome-calculator/template
@@ -3,6 +3,7 @@ pkgname=gnome-calculator
 version=3.34.1
 revision=1
 build_style=meson
+build_helper="gir"
 hostmakedepends="glib-devel itstool pkg-config vala"
 makedepends="gsettings-desktop-schemas-devel gtksourceview4-devel libmpc-devel
  libsoup-devel libgee08-devel"
```

**Comment by Piraty** [comment]
I have a gnome installation on a tablet, will try to test as well

**Comment by k4leg** [comment]
Yes, gnome-calculator does not cross-compile without `build_helper=gir`. Apparently I somehow did not notice it.
Thanks for the help!
Can you tell me how you do `diff --git`?

File: srcpkgs/evolution/template (hunk 1 above)
Comment (k4leg) [reply srcpkgs/evolution/template]: Ok. I will not do this anymore.

**Comment by Johnnynator** [comment]
> Can you tell me how you do `diff --git`?
I did just run `git diff` and pasted the output in code tags

**Comment by k4leg** [comment]
Thanks you! Now I am compiling `fractal` and `gnome-podcasts`, after the build I will check this.

**Comment by k4leg** [comment]
`fractal` and `gnome-podcasts` work fine on _x86_64-musl_. Thanks @Johnnynator!

**Comment by k4leg** [comment]
Is there any way to remove the `Merge branch 'master' into gnome [ci skip]`?

**Comment by Johnnynator** [comment]
> EDIT 2: How to remove conflicts without Resolve conflicts?

I would recommend to rebase on top of this repos master branch. so if you have `void-linux/void-packages` remote named as e.g. `upstream` it would be `git pull upstream master --rebase`

**Comment by k4leg** [comment]
@Johnnynator Thanks!

File: srcpkgs/empathy/template [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
+	hostmakedepends+=" telepathy-mission-control-devel glib-devel geoclue2-devel"
+	_commit=39f04a882154e82d43f211d18954401bb561fe12
+	distfiles+=" https://gitlab.gnome.org/GNOME/telepathy-account-widgets/-/archive/master/$_commit.tar.bz2?path=tp-account-widgets>taw-tpaw-$_commit.tar.bz2"
+	checksum+=" ac56d16592f5533fdf6a58be5536e34b319d4ecd48759653f019d48e20db24f6"
+
+	pre_build() {
+		mv ../telepathy-account-widgets-master-$_commit-tp-account-widgets/* .
+		rmdir ../telepathy-account-widgets-master-$_commit-tp-account-widgets
+	}
+fi
```
Comment (Piraty) [inline srcpkgs/empathy/template]: i hope you don't mind i fixed this mess in #17564

Comment (k4leg) [reply srcpkgs/empathy/template]: Thanks!

**Comment by k4leg** [comment]
It seems like `/usr/aarch64-linux-musl/usr/bin/g-ir-scanner` is missing and if so it would be necessary to remove it from `xbps_meson.cross`.

**Comment by romulasry** [comment]
Leave a message when this is committed, thanks.

**Comment by k4leg** [comment]
Everything is ready. I hope that I have not lost sight of anything.

**Comment by Hoshpak** [comment]
There are again conflicts. This PR is just to huge to review test and merge and I suspect it's also a burden to maintain it. Any chance you could break it down into smaller ones, a few packages at a time from bottom to top of the dependency chain?

File: srcpkgs/atk/template [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'atk'
 pkgname=atk
-version=2.34.1
+version=2.35.1
```
Comment (Johnnynator) [inline srcpkgs/atk/template]: This is afaik an beta version. (as all uneven ones)

File: srcpkgs/gnome-desktop/template [hunk 4]
```diff
@@ -9,14 +9,15 @@ configure_args="-Dgnome-distributor=VoidLinux -Dudev=enabled
 hostmakedepends="gettext itstool pkg-config glib-devel"
```
Comment (sgn) [inline srcpkgs/gnome-desktop/template]: `configure_args+=" -Dsystemd=disabled"`

File: srcpkgs/gnome-desktop/update [hunk 5]
```diff
@@ -1 +0,0 @@
-ignore="*.[0-9]*[13579].*"
```
Comment (sgn) [inline srcpkgs/gnome-desktop/update]: Do not delete this file.

**Comment by sgn** [comment]
Do NOT delete the file update

File: srcpkgs/libsoup/template [hunk 6]
```diff
@@ -1,27 +1,32 @@
...
-checksum=8308984f1eee1c4f8c113a9c1763b2b22d981bd811b0cc82a9f3f1aa63228779
+checksum=534bb08e35b0ff3702f3adfde87d3441e27c12f9f5ec351f056fe04cba02bafb
 
 # Package build options
 build_options="gir vala"
 build_options_default="gir vala"
 
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" -Dtls_check=false"
+	hostmakedepends+=" smbclient"
```
Comment (sgn) [inline srcpkgs/libsoup/template]: remove smbclient from makedepends, set it in hostmakedepends instead,

then remove this line.

File: srcpkgs/gtk-vnc/template [hunk 7]
```diff
@@ -1,106 +1,46 @@
...
-	short_desc+=" - GTK+2 port"
-	pkg_install() {
-		vmove "usr/lib/libgtk-vnc-1.0.so*"
-		if [ "$build_option_gir" ]; then
-			vmove usr/lib/girepository-1.0/GtkVnc-1.0.typelib
-		fi
-	}
-}
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" perl glib-devel vala"
```
Comment (sgn) [inline srcpkgs/gtk-vnc/template]: merge into original hostmakedepends,
on native build, they're required by some packages in makedepends.

We don't need this line.

File: srcpkgs/gtk-vnc/template [hunk 8]
```diff
@@ -1,106 +1,46 @@
...
-
-do_install() {
-	cd ${wrksrc}/gtk2-build
-	make DESTDIR=${DESTDIR} install
-
-	cd ${wrksrc}/gtk3-build
-	make DESTDIR=${DESTDIR} install
-}
-
-gtk2-vnc_package() {
```
Comment (sgn) [inline srcpkgs/gtk-vnc/template]: No gtk2-vnc anymore?

File: srcpkgs/nautilus/template [hunk 9]
```diff
@@ -17,7 +17,8 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=37ce2c16a610c589dcc7660f9092446465568e38e29bce6ed8c24f2e8e0077f3
+checksum=bb06e226b965f479a0b8668c19c8e1762b9d8f039af6817961cc7a8f09e6f8a7
+patch_args="-Np1"
```
Comment (sgn) [inline srcpkgs/nautilus/template]: I guess you forget to include `patches` directory.

As of it's now, nautilus requires pango>=1.44 for
`pango_attr_insert_hyphens_new`

**Comment by sgn** [comment]
libsigc++ and glibmm update broke atkmm

**Pull request closed without merging**